internal/plugincommon/consensus/threshold_test.go

```diff
@@ -0,0 +1,97 @@
+package consensus
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_GteFPlusOne(t *testing.T) {
+	type args struct {
+		f           int
+		val         int
+		expectedRes bool
+	}
+	testCase := []args{
+		{
+			f:           1,
+			val:         2,
+			expectedRes: true,
+		},
+		{
+			f:           4,
+			val:         5,
+			expectedRes: true,
+		},
+		{
+			f:           4,
+			val:         4,
+			expectedRes: false,
+		},
+	}
+
+	for _, tc := range testCase {
+		res := GteFPlusOne(tc.f, tc.val)
+		require.Equal(t, tc.expectedRes, res, "they should be equal")
```

Comment:
maybe use a clearer message instead of "they"

Reply:
deleted the string message as these validations are pretty straightforward